Tetrahedral_remeshing/include/CGAL/tetrahedral_remeshing.h

```diff
@@ -267,19 +269,24 @@ void tetrahedral_isotropic_remeshing(
 * @tparam CurveIndex is the type of the indices for feature curves.
 *            If `c3t3` has been generated using `CGAL::make_mesh_3()`, it must match
 *            the `Curve_index` type of the model of the `MeshDomainWithFeatures_3` concept used for mesh generation.
-* @tparam NamedParameters a sequence of \ref Remeshing_namedparameters "Named Parameters"
 *
-* @param c3t3 the complex containing the triangulation (of type `Tr`) to be remeshed.
-
-* @todo finish this doc
+* @param c3t3 the complex containing the triangulation to be remeshed.
+*
+* @todo mention `std::move(c3t3)`
 */
-template<typename Tr, typename CornerIndex, typename CurveIndex>
-typename Tr::Tr_Base
-convert_to_base_triangulation(
+
+template<typename Tr,
+         typename CornerIndex,
+         typename CurveIndex>
+CGAL::Triangulation_3<typename Tr::Geom_traits,
+                      typename Tr::Triangulation_data_structure>
+convert_to_triangulation_3(
```

Comment:
As I said, I prefer to mention the third template parameter: `Lock_data_structure`.

```suggestion
template<typename Tr,
         typename CornerIndex,
         typename CurveIndex>
CGAL::Triangulation_3<typename Tr::Geom_traits,
                      typename Tr::Triangulation_data_structure,
                      typename Tr::Lock_data_structure>
convert_to_triangulation_3(
```